tests/functional/lang/parse-fail-utf8.err.exp

```diff
@@ -1,4 +1,4 @@
-error: syntax error, unexpected invalid token, expecting end of file
+error: syntax error, unexpected invalid token
```

Comment:
I'm not sure why this error message changed, but it seems maybe not important? I'm not sure why we were expecting EOF before, as `123 + 4`, for example, is a valid parse.